app/routes/topics.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  model: function () {
+    return this.store.findAll('topic');
+  },
+
+  actions: {
+    createTopic (topic) {
+      let controller =  this.get('controller');
+      topic.save().then(function () {
+        controller.replaceNewTopic();
+      }, function () {});
```

Comment:
It's actually needed to not throw the error to the console :/  
I got it from http://alexspeller.com/server-side-validations-with-ember-data-and-ds-errors/
right under "Intro to DS.Errors"